src/ooni/http_invalid_request_line_impl.hpp

```diff
@@ -76,7 +76,13 @@ class HTTPInvalidRequestLineImpl : public TCPTestImpl {
     void main(Settings options, std::function<void(json)> &&cb) {
         entry["tampering"] = nullptr;
 
-        http::Url backend_url = mk::http::parse_url(options["backend"]);
+        ErrorOr<http::Url> backend_url = mk::http::parse_url_noexcept(options["backend"]);
+
+        if (!backend_url) {
+            std::cout << "hey" << std::endl;
```

Comment:
This debugging information should not be merged into master. Could you remove it?